src/scene-entry-manager.js

```diff
@@ -195,7 +196,46 @@ export default class SceneEntryManager {
     el.setAttribute("networked", { persistent: true });
     el.setAttribute("media-loader", { fileIsOwned: true });
 
-    this.hubChannel.pin(networkId, gltfNode, fileId, fileAccessToken, promotionToken);
+    try {
+      await this.hubChannel.pin(networkId, gltfNode, fileId, fileAccessToken, promotionToken);
+    } catch (e) {
+      if (e.reason === "invalid_token") {
+        await this.authChannel.signOut(this.hubChannel);
```

Comment:
currently the hub page load ignores `invalid_tokens` and just chugs along -- you might want to handle that properly too now. iirc the reason i punted was because we were in incident response and I wasn't sure how to properly update all the UI state, etc.